libpod/container_internal.go

```diff
@@ -676,7 +676,7 @@ func (c *Container) getAllDependencies(visited map[string]*Container) error {
 	}
 	for _, depID := range depIDs {
 		if _, ok := visited[depID]; !ok {
-			dep, err := c.runtime.state.LookupContainer(depID)
+			dep, err := c.runtime.state.Container(depID)
```

Comment:
Should this be GetContainer?

Reply:
The runtime method is GetContainer(), but the state method is called Container(). This is especially confusing because all GetContainer does is call state.Container() with the same argument.

Reply:
OK, thx.  Personally I think I'd vote to change this to c.runtime.GetContainer(depID) as it's clearer what's going on.  Even though you get that extra function hop.  But shrug.

Reply:
The recommended method is the exact opposite.

GetContainer -> Container

Go standards, I believe state that the functions should be SetFoobar() and Foobar()

Reply:
Yeah, it's the one thing that I really miss from my java days.